templates/phaser/mprocs.yaml

```diff
@@ -8,3 +8,11 @@ procs:
   anvil:
     cwd: packages/contracts
     shell: anvil --base-fee 0 --block-time 2
+  indexer:
+    shell: rimraf $SQLITE_FILENAME && pnpm sqlite-indexer
+    env:
+      RPC_HTTP_URL: "http://127.0.0.1:8545"
+      FOLLOW_BLOCK_TAG: "latest"
+      SQLITE_FILENAME: "indexer.db"
+  explorer:
+    shell: pnpm explorer --worldsFile packages/contracts/worlds.json
```

Comment:
I am curious if it would make more sense to run this and indexer in the context of the contracts package
```
indexer:
  cwd: packages/contracts
...
explorer:
  cwd: packages/contracts
  shell: pnpm explorer --worldsFile worlds.json
```

Reply:
Like this idea just so there are no flags as part of the default setup, will make the change